### PR TITLE
[FLINK-12749] [playgrounds] initial version of flink-cluster-playground

### PR DESCRIPTION
**Pull request by knaufk**
see https://github.com/apache/flink/pull/9192

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+processing capabilities.
+
+Learn more about Flink at [http://flink.apache.org/](http://flink.apache.org/)
+
+## Playgrounds
+
+This repository contains the configuration files for two Apache Flink playgrounds.
+
+* The [Flink Cluster Playground](../master/flink-cluster-playground) consists of a Flink Session Cluster, a Kafka Cluster and a simple 
+Flink Job. It is explained in much detail as part of
```
Comment (sjwiesman) [inline README.md]:
```suggestion
Flink Job. It is explained in detail as part of 
```

**Review by sjwiesman — COMMENTED** [review]
One small grammar fix but otherwise looks good. However, I would prefer we not merge this until the corresponding docs are merged in in-case we need to make any changes.

File: flink-cluster-playground/docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    ports:
+      - 8081:8081
+    volumes:
+      - ./conf:/opt/flink/conf
+      - flink-checkpoint-directory:/tmp/flink-checkpoint-directory
+      - /tmp/flink-savepoints-directory:/tmp/flink-savepoints-directory
+    environment:
+      - JOB_MANAGER_RPC_ADDRESS=jobmanager
+  taskmanager:
+    image: flink:1.9-SNAPSHOT-scala_2.11
```
Comment (fhueske) [inline flink-cluster-playground/docker-compose.yaml]: rm `SNAPSHOT`

File: README.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+* The [Flink Cluster Playground](../master/flink-cluster-playground) consists of a Flink Session Cluster, a Kafka Cluster and a simple 
+Flink Job. It is explained in much detail as part of 
+[Apache Flink's "Getting Started" guide](https://ci.apache.org/projects/flink/flink-docs-stable/getting-started/docker-playgrounds/flink-cluster-playground.html). 
+
+* The interactive SQL playground is still under development and will be added shortly.
+
+## About
+
+Apache Flink is an open source project of The Apache Software Foundation (ASF).
+The Apache Flink project originated from the [Stratosphere](http://stratosphere.eu) research project.
```
Comment (fhueske) [inline README.md]: I think we can remove this line ;-)

**Review by fhueske — COMMENTED** [review]
Thanks for the PR!
Just found two minor things.

Cheers, Fabian

**Comment by sjwiesman** [comment]
@fhueske are there release steps documented somewhere? This repo might not have a lot of visibility within the flink-dev community, and we should make sure release managers are aware of it.

**Comment by fhueske** [comment]
Hi @sjwiesman, the release process is documented in the wiki: https://cwiki.apache.org/confluence/display/FLINK/Creating+a+Flink+Release

File: README.md (hunk 3 above)
Comment (knaufk) [reply README.md]: We could also remove it from the `flink/README.md`?

File: flink-cluster-playground/conf/flink-conf.yaml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+jobmanager.rpc.address: jobmanager
+blob.server.port: 6124
+query.server.port: 6125
+
+taskmanager.numberOfTaskSlots: 2
+
```
Comment (fhueske) [inline flink-cluster-playground/conf/flink-conf.yaml:24]: Add the following lines to improve responsiveness in the failure simulation:

```
heartbeat.interval: 1000
heartbeat.timeout: 5000
```

Comment (knaufk) [reply flink-cluster-playground/conf/flink-conf.yaml:24]: Thanks! Done.

**Pull request merged**